### ISSUE TITLE
How can i manage with imbalance class

### ISSUE THREAD

**Issue by 5730279821-TA** [matterport/Mask_RCNN#1051]
I want to train Mask R-CNN with my dataset that have 4 classes. 
Class A -> 3,000
Class B -> 26
Class C -> 8
Class D -> 4

Model always predict Class A because my data was imbalance
How can i fix this problem ?.

**Comment by lovedanang** [comment]
I recommend 2 ways. first way and best way, you should find more data for B,C and D. second way, you could try class_weight [https://keras.io/models/sequential/](url)

**Comment by fastlater** [comment]
The difference is huge. Maybe using augmentation can give you more samples but as @lovedanang suggested, find more data for those classes. Also, you can try something different than deep learning if you dataset is that small.
